io.home-assistant.timestamp/manifest.tt

```diff
@@ -0,0 +1,13 @@
+abstract class @io.home-assistant.timestamp
+#_[thingpedia_name="Timestamp Sensor"]
+#_[thingpedia_description="Interface for Home Assistant's Timestamp Sensor."]
+#_[description="Interface for Home Assistant's Timestamp Sensor."]
+#[license="CC-0"]
+#[license_gplcompatible=true]
+#[subcategory="home"]
+{
+  monitorable query state(out value : String)
+  #_[canonical="timestamp sensor state"]
+  #_[confirmation="the state of your timestamp sensor"]
+  #_[formatted=["The timestamp is ${value}."]];
```

Comment:
What's a timestamp sensor?
This greatly smells like something we absolutely don't want in Almond.

Reply:
LOL yeah we weren't sure about this one. Timestamp is one of the device classes for sensor - https://developers.home-assistant.io/docs/en/entity_sensor.html but there wasn't a demo device for this so I'm not exactly sure what it outputs.

Reply:
Anything we are not sure about, let's drop for now.